app/models/ems_refresh/parsers/openstack.rb

```diff
@@ -65,23 +65,23 @@ def ems_inv_to_hashes
     private
 
     def servers
-      @servers ||= @connection.servers_for_accessible_tenants
+      @servers ||= @connection.handled_list(:servers)
```

Comment:
I'm still on the fence about `@connection.handle_list(:servers)` vs. `@connection.servers` that then calls `handled_list(:servers)` internally (or the suggested name below: `list(:servers)`).

@Fryguy, @matthewd, thoughts here?